src/main/java/com/commercetools/sync/taxcategories/helpers/TaxCategoryDraftBuilderHelper.java

```diff
@@ -0,0 +1,30 @@
+package com.commercetools.sync.taxcategories.helpers;
+
+import io.sphere.sdk.taxcategories.TaxCategory;
+import io.sphere.sdk.taxcategories.TaxCategoryDraft;
+import io.sphere.sdk.taxcategories.TaxCategoryDraftBuilder;
+import io.sphere.sdk.taxcategories.TaxRateDraft;
+import io.sphere.sdk.taxcategories.TaxRateDraftBuilder;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaxCategoryDraftBuilderHelper {
```

Comment:
It is not possible to easily create a TaxCategoryDraft instance from a TaxCategory instance. There is no method for this in the class _TaxCategoryDraftDraftBuilder_, for example _ProductTypeDraftBuilder_ has such a method: _public static ProductTypeDraftBuilder of(final ProductType template)_
We had a few cases where such conversion was needed, so this class was created. I thought it would be good if it was generally available. The best solution would be to add this method to _TaxCategoryDraftBuilder_ but I don't know if it is possible. The same applies to the _StateDraftBuilderHelper_ class.
If you think this is not a right place - we can remove it.